wkcuber/tests/test_cli.py

```diff
@@ -1,715 +1,165 @@
 
-    assert (tmp_path / "color").is_dir()
-    assert (tmp_path / "color" / "1").is_dir()
-    assert count_wkw_files(tmp_path / "color" / "1") == 1
-    assert (tmp_path / PROPERTIES_FILE_NAME).exists()
-
-
-def test_tiff_formats_cubing_I(tmp_path: Path) -> None:
-    check_call(
-        "python",
-        "-m",
-        "wkcuber.convert_image_stack_to_wkw",
-        "--jobs",
-        2,
-        "--batch_size",
-        8,
-        "--layer_name",
-        "color",
-        "--max_mag",
-        4,
-        "--target_mag",
-        2,
-        "--voxel_size",
-        "1,1,1",
-        "--name",
-        "awesome_data",
-        "--no_compress",
-        "--chunks_per_shard",
-        8,
-        TESTDATA_DIR / "various_tiff_formats" / "test_I.tif",
-        tmp_path,
-    )
+        result_downsample = runner.invoke(app, ["downsample", str(wkw_path)])
 
-    assert (tmp_path / "color").is_dir()
-    assert (tmp_path / "color" / "2").is_dir()
-    assert count_wkw_files(tmp_path / "color" / "2") == 1
-    assert (tmp_path / PROPERTIES_FILE_NAME).exists()
-
-
-def test_tiff_formats_cubing_C(tmp_path: Path) -> None:
-    # to make output wk_compatible, these files require multiple layer
-    check_call(
-        "python",
-        "-m",
-        "wkcuber.convert_image_stack_to_wkw",
-        "--jobs",
-        2,
-        "--batch_size",
-        8,
-        "--layer_name",
-        "color",
-        "--max_mag",
-        1,
-        "--voxel_size",
-        "1,1,1",
-        "--name",
-        "awesome_data",
-        "--no_compress",
-        "--chunks_per_shard",
-        8,
-        TESTDATA_DIR / "various_tiff_formats" / "test_C.tif",
-        tmp_path,
-    )
+        assert result_downsample.exit_code == 0
+        assert (wkw_path / "color" / "1" / "z0" / "y0" / "x0.wkw").exists()
+        assert (wkw_path / "color" / "2" / "z0" / "y0" / "x0.wkw").exists()
+        assert (wkw_path / "color" / "4" / "z0" / "y0" / "x0.wkw").exists()
 
-    assert (tmp_path / "color_0").is_dir()
-    assert (tmp_path / "color_1").is_dir()
-    assert (tmp_path / "color_2").is_dir()
-    assert (tmp_path / "color_3").is_dir()
-    assert (tmp_path / "color_4").is_dir()
-    assert (tmp_path / "color_0" / "1").is_dir()
-    assert count_wkw_files(tmp_path / "color_0" / "1") == 1
-    assert (tmp_path / PROPERTIES_FILE_NAME).exists()
-
-
-def test_tiff_formats_cubing_S(tmp_path: Path) -> None:
-    check_call(
-        "python",
-        "-m",
-        "wkcuber.convert_image_stack_to_wkw",
-        "--jobs",
-        2,
-        "--batch_size",
-        8,
-        "--layer_name",
-        "color",
-        "--max_mag",
-        1,
-        "--voxel_size",
-        "1,1,1",
-        "--name",
-        "awesome_data",
-        "--no_compress",
-        "--chunks_per_shard",
-        8,
-        TESTDATA_DIR / "various_tiff_formats" / "test_S.tif",
-        tmp_path,
-    )
+        Dataset.open(wkw_path).get_layer("color").delete_mag(1)
```

Comment:
You probably delete the mag 1 from a copy of the dataset, because other tests depend on it.